### PR TITLE
update kapacitor to the latest release and some fixes to match best practices

### PR DESCRIPTION
**Pull request by cpanato**
**What this PR does / why we need it**:
- Update the chart to use the latest release
- fix the volume claim to accept an existing one
- fix the username in the chart.yaml
- bump chart version to 1.0.0 since it is stable

### CODE REVIEW

File: stable/kapacitor/templates/pvc.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-{{- if .Values.persistence.enabled }}
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+{{- if .Values.influxURL }}
```
Comment (davidkarlsen) [inline stable/kapacitor/templates/pvc.yaml:2]: why the influxURL?

File: stable/kapacitor/templates/pvc.yaml [hunk 2]
```diff
@@ -1,4 +1,5 @@
-{{- if .Values.persistence.enabled }}
```
Comment (davidkarlsen) [inline stable/kapacitor/templates/pvc.yaml:1]: why remove this?

Comment (cpanato) [reply stable/kapacitor/templates/pvc.yaml:1]: not removing adding another test
`{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}`

File: stable/kapacitor/templates/pvc.yaml (hunk 1 above)
Comment (cpanato) [reply stable/kapacitor/templates/pvc.yaml:2]: following the same approach here: https://github.com/helm/charts/blob/master/stable/kapacitor/templates/deployment.yaml#L1

if you not set the influxURL the deployment will not be created. and if the deployment is not created let's apply the same for the PVC, to not have a not used resource hanging around

**Comment by cypressious** [comment]
@cpanato This update sets the image version to 1.6.2. As far as I can tell, such a version of Kapacitor doesn't exist.

**Comment by cpanato** [comment]
@cypressious looks like it was removed otherwise the PR will not get merged.

I just checked the docker hub and it disappears. I will update to 1.5-alpine

**Comment by cpanato** [comment]
PR: https://github.com/helm/charts/pull/7790 @cypressious

**Pull request merged**